internal/automation-scripts/BuildStaticScanZip.sh

```diff
@@ -48,7 +48,21 @@ do
 				find $STATIC_HOME/$product -regex $actual_pattern -exec cp {} $STATIC_HOME/$product-work \;
 			else
 				echo "$SCRIPT_TAG Copying files that match the pattern: $pattern"
-				find $STATIC_HOME/$product -name $pattern -exec cp {} $STATIC_HOME/$product-work \;
+				#Find and Copy all the js,jsp and jaggery files to scan artifact
+				if [[ $pattern == *".js"* ]] || [[ $pattern == *".jag"* ]] ;then
+            		for file in $(find $STATIC_HOME/$product -name $pattern)
```

Comment:
Shall we correct the indentations here? This should be nested inside the else?

Reply:
Resolved